ui/index.d.ts

```diff
@@ -1112,6 +1113,26 @@ export function createAzureSubscriptionClient<T extends IAddUserAgent>(
  */
 export function createApiProvider(azExts: AzureExtensionApi[]): AzureExtensionApiProvider;
 
+/**
+ * Wraps the vscode.OutputChannel to add timestamps to all messages by calling appendLog.
```

Comment:
I would keep the docs for the class more generic and move all of the comments specific to `appendLog` down onto `appendLog`